enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -23,9 +23,11 @@
 
 (use-fixtures :once (fixtures/initialize :test-users))
 
+(def ^:private other-enabled-premium-features #{:audit-app})
+
 (defn- disable-other-sso-types [thunk]
   (let [current-features (premium-features/token-features)]
-    (premium-features-test/with-premium-features #{:sso-saml}
+    (premium-features-test/with-premium-features (set/union #{:sso-saml} other-enabled-premium-features)
```

Comment:
This can use `with-additional-premium-features` as well, then you don't need `other-enabled-premium-features`

Reply:
Usually that's a good idea, however I don't think it is in this case. I think `disable-other-sso-types` is meant to silence other premium features. That's why I handled this one a little differently.

Reply:
We chatted some more, and I'm going to try skipping the 2 premium features setting macros here.